pycheribuild/projects/cross/compiler-rt-builtins.py

```diff
@@ -0,0 +1,102 @@
+#
```

Comment:
I would be tempted to merge these two files since they are building the same project, just different subsets.